js/index.js

```diff
@@ -73,6 +73,11 @@
 
   // renderers
 
+  function setTitle (title) {
```

Comment:
This isn't necessarily needed since the`<h1 class="gallery-title"></h1>` won't be merged in, but let's keep it since we might use it later on.